src/app/clusters/camera-av-stream-management-server/camera-av-stream-management-server.cpp

```diff
@@ -568,19 +551,19 @@ CHIP_ERROR CameraAVStreamMgmtServer::Read(const ConcreteReadAttributePath & aPat
         break;
     case ImageRotation::Id:
         VerifyOrReturnError(
-            SupportsOptAttr(OptionalAttribute::kImageRotation), CHIP_IM_GLOBAL_STATUS(UnsupportedAttribute),
+            HasFeature(Feature::kImageControl), CHIP_IM_GLOBAL_STATUS(UnsupportedAttribute),
```

Comment:
Optional feature and so should have the Optional attribute check here, no? The check against the Feature bit should already have been done in Init().

Reply:
Yep, fixed.